denops/@ddu-ui-ff/preview.ts

```diff
@@ -0,0 +1,317 @@
+import {
+  ActionFlags,
+  BufferPreviewer,
+  Context,
+  DduItem,
+  DduOptions,
+  NoFilePreviewer,
+  PreviewContext,
+  Previewer,
+  TermPreviewer,
+} from "../../../ddu.vim/denops/ddu/types.ts";
```

Comment:
You cannot use relative path.

Reply:
Yes, it is for temporary use. So, can you update the ddu version to 1.6.0?